GunShipAlpha/src/Enemy.java

```diff
@@ -162,3 +162,33 @@ private ArrayList<ExplosionParticle> createExplosionParticles() {
         return particles;
     }
 }
+
+class Ginker extends Enemy{
+
+    public Ginker(int x, int y, int targetX, int targetY) {
+        super(x, y, targetX, targetY);
+        this.sprites = new GinkerSprite();
+    }
+    @Override
+    public void moveTowardTarget() {
```

Comment:
i want to see how this works in practice, but i do like the way that the subclass arrangement shows how each enemy type differs from the others quite clearly. neat!